include/SDL3/SDL_gpu.h

```diff
@@ -1729,7 +1776,7 @@ typedef struct SDL_GPUColorTargetInfo
     SDL_GPUTexture *texture;         /**< The texture that will be used as a color target by a render pass. */
     Uint32 mip_level;                /**< The mip level to use as a color target. */
     Uint32 layer_or_depth_plane;     /**< The layer index or depth plane to use as a color target. This value is treated as a layer index on 2D array and cube textures, and as a depth plane on 3D textures. */
-    SDL_FColor clear_color;          /**< The color to clear the color target to at the start of the render pass. Ignored if SDL_GPU_LOADOP_CLEAR is not used. */
+    SDL_GPUClearColor clear_color;   /**< The color to clear the color target to at the start of the render pass. Ignored if SDL_GPU_LOADOP_CLEAR is not used. */
```

Comment:
@slouken Dumb question: does going from essentially `float[4]` to `union { float[4], int[4], uint[4] }` count as an ABI break?

Reply:
If it doesn't change the structure size or alignment, this is probably fine. The clear color won't work correctly if someone uses the old API on integer formats, but it didn't work correctly before, so that's okay.